### PR TITLE
[RFC][NI] Update step card

### PR DESCRIPTION
**Pull request by henrmota**
https://uniplaces.atlassian.net/browse/UN-9887

### CODE REVIEW

File: src/styles/_components/_step-card.scss [hunk 1]
```diff
@@ -8,8 +8,9 @@
     min-height: 210px;
     padding: 32px;
     margin-right: 20px;
-    background-color: color('background', 'modal');
-    
+    background-color: rgba(color('primary', 'primary'), 0.15);
```
Comment (mmelo) [inline src/styles/_components/_step-card.scss]: use double quotes, please

File: src/styles/_components/_step-card.scss [hunk 2]
```diff
@@ -22,18 +23,18 @@
 
     &__step {
         width: 100%;
-        margin-bottom: 8px;
+        margin-bottom: 50px;
         @include font("IntervalNextReg");
-        font-size: em(12px);
-        color: color('text', 'subtitle');
+        font-size: em(20px);
+        color: color('primary', 'primary');
```
Comment (mmelo) [inline src/styles/_components/_step-card.scss]: same here

File: src/styles/_components/_step-card.scss [hunk 3]
```diff
@@ -42,15 +43,15 @@
         margin-bottom: 14px;
         @include font("IntervalNextBook");
         font-size: em(16px);
-        color: color('text', 'subheading');
+        color: color('text', 'header');
```
Comment (mmelo) [inline src/styles/_components/_step-card.scss]: same here please

File: src/styles/_components/_step-card.scss [hunk 4]
```diff
@@ -64,7 +65,11 @@
 
     &--disabled {
         background-color: transparent;
-        
+
+        .uni-step-card__step{
+            color: color('text', 'subtitle');
```
Comment (mmelo) [inline src/styles/_components/_step-card.scss]: same here

File: src/styles/_components/_step-card.scss [hunk 5]
```diff
@@ -22,38 +23,38 @@
 
     &__step {
         width: 100%;
-        margin-bottom: 8px;
+        margin-bottom: 50px;
         @include font("IntervalNextReg");
-        font-size: em(12px);
-        color: color('text', 'subtitle');
+        font-size: em(20px);
+        color: color("primary", "primary");
```
Comment (AndreJoaquim) [inline src/styles/_components/_step-card.scss]: You don't need to add the second key if it's `primary` as the second key defaults to `primary`.

```sass
color: color("primary");
```

File: src/styles/_components/_step-card.scss [hunk 6]
```diff
@@ -64,7 +65,11 @@
 
     &--disabled {
         background-color: transparent;
-        
+
```
Comment (AndreJoaquim) [inline src/styles/_components/_step-card.scss]: Remove empty line

File: src/styles/_components/_step-card.scss [hunk 7]
```diff
@@ -64,7 +65,11 @@
...
+        .uni-step-card__step{
```
Comment (AndreJoaquim) [inline src/styles/_components/_step-card.scss]: Missing space

File: src/styles/_components/_step-card.scss [hunk 8]
```diff
@@ -8,8 +8,9 @@
     min-height: 210px;
     padding: 32px;
     margin-right: 20px;
-    background-color: color('background', 'modal');
-    
+    background-color: rgba(color("primary", "primary"), 0.15);
```
Comment (AndreJoaquim) [inline src/styles/_components/_step-card.scss]: `color("primary")`

**Pull request merged**